cime_config/machines/config_machines.xml

```diff
@@ -3096,7 +3096,7 @@
     <RUNDIR>$CIME_OUTPUT_ROOT/$CASE/run</RUNDIR>
     <EXEROOT>$CIME_OUTPUT_ROOT/$CASE/bld</EXEROOT>
     <environment_variables compiler="intel">
-      <env name="NETCDF_PATH">/usr/gdata/e3sm/install/quartz/netcdf-fortran/</env>
+      <env name="NETCDF_PATH">/usr/workspace/e3sm/install/quartz/netcdf-fortran/</env>
```

Comment:
I think the next step will be to redo this install, so maybe we can leave it like this for now.  But having `quartz` in the name of the path I think would be confusing, since quartz no longer exists.

Reply:
We'll leave this for the moment. There will be another update to use the new modules I'm building once I sort out the last couple issues.